savoirfairelinux/num2words
# changed ktiv haser to ktiv male (long form when not using diacritics)

## Fixes # by Eyal Gruss

### Changes proposed in this pull request:

fixed short forms (ktiv haser) which are relevant when using hebrew diacritics, to long-form (ktiv male) which is the formal way the write without diacritics. Also some of the short-forms were outdated biblical forms

### Status


### How to verify this change

search it here: https://hebrew-academy.org.il/ and observe the form labeled "ללא ניקוד" (=without diacritics)


### Additional notes

This explains the difference between the two systems and states the long-form is the formal one:
https://hebrew-academy.org.il/topic/hahlatot/missingvocalizationspelling/

you may also want to support the short form with the following considerations:
1. they should appear with diacritics (and this will require changing the logic for the hundreds as the diacritics defer due to construct (סמיכות) form)
2. it should be a secondary option, with the long form being the default
3. do not use biblical forms e.g שלש
